### PR TITLE
Add insecure option to MinIO driver

### LINKED ISSUE #359: Add `-insecure` option to MinIO driver
**Describe the Issue (Bug/Feature)**

At the moment, driver-minio does not assume endpoints with self-signed certificates, and this causes exceptions when we try we try to https endpoints for these S3 compatible boxes.

**To Reproduce**

1. Prepare a S3 compatible object storage with a self-signed certificate.
2. Invoke SBK with `-class minio` and other workload specific parameters to the S3 compatible box.

**Expected behavior**

Disabling certificate validation and continue performance measurements.

Or, I think it's also worth considering adding '-cacert' option, but it should be another issue.

**Additional context**

**Note**
I already have a working patch. If this looks reasonable for the community, I can contribute it.

### PR DESCRIPTION
**Pull request by thatsdone**
Add '-insecure' option to MinIO driver, and resolve #359

Signed-off-by: Masanori Itoh <masanori.itoh@gmail.com>

### CODE REVIEW

File: driver-minio/src/main/resources/minio.properties [hunk 1]
```diff
@@ -20,3 +20,6 @@ bucketName=sbk
 
 # Recreate the bucket; if already existing , delete it and create it again
 reCreate=false
+
+# No server certificate validation
+insecure=false
```
Comment (kmgowda) [inline driver-minio/src/main/resources/minio.properties:25]: by default , it should be "true"; so that our regular tests works

Comment (kmgowda) [inline driver-minio/src/main/resources/minio.properties:25]: by default , it should be "true"; so that our regular tests works

Comment (thatsdone) [reply driver-minio/src/main/resources/minio.properties:25]: BTW, `-insecure` option has no effect for non-TLS/SSL endpoint. Actually, I'm measuring non TLS/SSL S3 box performance without `-insecure` option (that is equivalent to `-insecure false'). Also, IMHO, if we add TLS/SSL boxes with self-signed certificates for our regular testing, test cases should be rewritten (with `-insecure true').

But, anyway, I will update my PR.

**Comment by kmgowda** [comment]
@thatsdone thank you ..

**Comment by kmgowda** [comment]
@thatsdone  is it possible update this driver with TLS authentication and security? you need add additional params such -key  and -cert
not a manditory , in case you are using minio server with TLS enabled or any other security enabled , then i suggest the update the sbk-minio driver too.

**Pull request merged**